state/src/storage/tx/node_metadata.rs

```diff
@@ -94,7 +95,7 @@ impl<'db, T: TransactionKind> StateTxn<'db, T> {
     }
 
     /// Get the local relayer's fee decryption key
-    pub fn get_fee_decryption_key(&self) -> Result<DecryptionKey, StorageError> {
+    pub fn get_fee_key(&self) -> Result<RelayerFeeKey, StorageError> {
         self.inner()
             .read(NODE_METADATA_TABLE, &LOCAL_RELAYER_DECRYPTION_KEY.to_string())?
```

Comment:
nit: can we rename `LOCAL_RELAYER_DECRYPTION_KEY` to `LOCAL_RELAYER_FEE_KEY`?

Reply:
yes, good catch